xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -2533,7 +2439,7 @@ xe.userdirectory.canCustomizeInfo=The user directory can be [[customized>>{0}||q
 xe.userdirectory.canCustomizeInfoGuest=The user directory can be customized to display the columns you wish to see, but you need to [[log in>>{0}]] first.
 xe.userdirectory._avatar=Picture
 xe.userdirectory.doc.name=User ID
-xe.userdirectory.emptyvalue=
+### Missing: xe.userdirectory.emptyvalue=
```

Comment:
This is not good in the source property file.

Reply:
And not only that, weblate cleaned a lot of empty lines, which is not its job. It's like it tried to regenerate this file as a translation instead of the source file.

Reply:
> And not only that, weblate cleaned a lot of empty lines, which is not its job. It's like it tried to regenerate this file as a translation instead of the source file.

So you have to be aware that there's no dedicated mechanism for source file in weblate code right now: we always re-generate the entire file from the list of translations. I haven't provided a specific implem for source files. So those changes makes sense from that point of view.